public/static/locales/en/me.json

```diff
@@ -194,5 +194,13 @@
     "stripe-sepa_debit": "SEPA payments take 2-14 business days (usually 3-5 days). As soon as your transfer arrives we will send you a confirmation email with your receipts (and your certificate).",
     "stripe-sofort": "SOFORT payments take 2-14 business days (usually 2-3 days). As soon as your transfer arrives we will send you a confirmation email with your receipts (and your certificate).",
     "in-dispute": "Your transfer was disputed. Usually $7 - $15.00 dispute fee is charged by Payment Providers. Please Contact support@plant-for-the-planet.org for details."
-  }
+  },
+  "active": "active",
+  "canceled": "canceled",
+  "incomplete": "incomplete",
+  "incomplete_expired": "incomplete & expired",
+  "past_due": "past due",
+  "unpaid": "unpaid",
+  "trialing": "trialing",
+  "paused": "paused"
```

Comment:
For the user, we only show
Active --> active/trialing
Pending --> Incomplete, 
Cancelled --> incomplete/expired/canceled
Paused -> paused

Reply:
Ok. That could easily be done by changing the translations.

Reply:
So I am assuming this PR is good to merge

Reply:
I will change the resources in this PR as following (so we do not have to wait for a translation run):

```
  "active": "Active",
  "canceled": "Cancelled",
  "incomplete": "Pending",
  "incomplete_expired": "Cancelled",
  "past_due": "Past due",
  "unpaid": "Unpaid",
  "trialing": "Active",
  "paused": "Paused"
```

As @sagararyal  did not say anything about unpaid or past_due I keep them.